.circleci/config.yml

```
@@ -73,6 +74,21 @@ jobs:
       - run: cd build && cmake --build . -j $(nproc)
       - store_artifacts: {path: ./build/devilutionx.nro, destination: devilutionx.nro}
 
+  3ds:
+    docker:
+      - image: devkitpro/devkitarm:latest
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: echo deb http://deb.debian.org/debian stretch-backports main > /etc/apt/sources.list.d/debian-backports.list
+      - run: apt-get update && apt-get install -y -t stretch-backports cmake
+      # Install cmake files (https://github.com/devkitPro/docker/issues/3)
+      - run: dkp-pacman -S --needed --noconfirm --quiet devkitpro-pkgbuild-helpers
```

You might need to update the devkitpro installation first, as was done recelty for the switch section:
```suggestion
      - run: dkp-pacman -Syu --noconfirm
      - run: dkp-pacman -S --needed --noconfirm --quiet devkitpro-pkgbuild-helpers
```